### ISSUE TITLE
Receive address verification on secondary/offline PC

### ISSUE THREAD

**Issue by tschaboo** [JWWeatherman/yeticold#169]
Is there a possiblity to check receive addresses on the offline PC?

While it's unlikely to immediately get malware on a fresh Ubuntu install, I think it's still a threat.

**Comment by rollomcfloogle** [comment]
Yes, you can. Open the Console (Window > Console) and use the getaddressinfo command:

getaddressinfo "address"

The ismine field will tell you if the address is the wallet's (true) or not (false).
